src/rest-api/endpoints/management-endpoint/binaries-endpoint.md

```diff
@@ -11,6 +11,11 @@ toc: true
 tree_item_index: 100
 ---
 
+{{#> callout type='warning' heading='Deprecated Garbage Collection'}}
+This page refers to an old Garbage Collection implementation known to not scale on big repositories. Above a certain number of documents/binaries, this implementation fails in timeout or OutOfMemory errors.
```

Comment:
maybe:
For MongoDB there is a new implementation that solves scalability problems, when possible use the new endpoint:  link